todoApp/src/components/DisplayTodo.tsx

```diff
@@ -0,0 +1,31 @@
+//Stateful component
+import React, { useState } from "react";
+import { FiCheckCircle } from "react-icons/fi"; //React icons
+import styles from "./DisplayTodo.module.css";
+
+interface TodoListsInterface {
+  title: string;
+}
+
+const DisplayTodo = (props: TodoListsInterface) => {
+  const [todoTask, setTodoTask] = useState(true);
+  const changeStatusHandler = () => {
```

Comment:
Just add 1 empty line before handler function and return statement.
Rest code is perfect and you can merge after doing this change.